pkg/server/deploy/deploy.go

```diff
@@ -22,6 +22,7 @@ const (
 type Operations interface {
 	Deploy(user *database.User, appName string, tarBall io.ReadSeeker, description string, opts *Options) (io.ReadCloser, error)
 	List(user *database.User, appName string) ([]*ReplicaSetListItem, error)
+	Rollback(user *database.User, rollback *Rollback) error
```

Comment:
Why did you choose to create a new struct instead of using simple strings? (I took a look at that struct and looks like very simple)

Reply:
Just to group all rollback parameters

Reply:
sounds unnecessary, but :shipit:

Reply:
:diego5: